### DISCUSSION TITLE
https://mnfienen.github.io/talulat/2023/08/25/day24.html?fbclid=IwAR14DMAQPdo9VYBGBc6ylHEJjH3qy0SUbYdehxhDdQyAa6JfeQvx65J750Y_aem_ARljQCkedWv_ALY7PyptDpzxEiM8xIix2DFVARHfOotjapR27vyvgduqumioLnDC1ck

### DISCUSSION THREAD

**Discussion by giscus** [mnfienen/talulat#34] [Comments]
# https://mnfienen.github.io/talulat/2023/08/25/day24.html?fbclid=IwAR14DMAQPdo9VYBGBc6ylHEJjH3qy0SUbYdehxhDdQyAa6JfeQvx65J750Y_aem_ARljQCkedWv_ALY7PyptDpzxEiM8xIix2DFVARHfOotjapR27vyvgduqumioLnDC1ck

Day 24 - The last day! Mixed emotions, for sure.

https://mnfienen.github.io/talulat/2023/08/25/day24.html?fbclid=IwAR14DMAQPdo9VYBGBc6ylHEJjH3qy0SUbYdehxhDdQyAa6JfeQvx65J750Y_aem_ARljQCkedWv_ALY7PyptDpzxEiM8xIix2DFVARHfOotjapR27vyvgduqumioLnDC1ck

**Comment by SmittysGitHub** [comment]
You are amazing!  The blog is fabulous!!

**Reply by mnfienen** [reply]
thanks for keeping us company!
